From f541550358f136e9a6d1fd131e83d17e6269dae4 Mon Sep 17 00:00:00 2001
From: Marcel Ritter <marcel@linux-ng.de>
Date: Tue, 21 Jun 2022 09:23:22 -0400
Subject: [PATCH] svcgssd: Add (undocumented) config options to man page

There seem to be some undocumented options implemented.
Why not mention them in the man page?

Signed-off-by: Steve Dickson <steved@redhat.com>
---
 utils/gssd/svcgssd.man | 13 +++++++++++++
 1 file changed, 13 insertions(+)

Origin: upstream, http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=commit;h=f541550358f136e9a6d1fd131e83d17e6269dae4
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/nfs-utils/+bug/1977745
Last-Update: 2022-06-27

diff --git a/utils/gssd/svcgssd.man b/utils/gssd/svcgssd.man
index 15ef4c9..8771c03 100644
--- a/utils/gssd/svcgssd.man
+++ b/utils/gssd/svcgssd.man
@@ -61,6 +61,19 @@ this is equivalent to the
 option.  If set to any other value, that is used like the
 .B -p
 option.
+.TP
+.B verbosity
+Value which is equivalent to the number of
+.BR -v .
+.TP
+.B rpc-verbosity
+Value which is equivalent to the number of
+.BR -r .
+.TP
+.B idmap-verbosity
+Value which is equivalent to the number of
+.BR -i .
+
 
 .SH SEE ALSO
 .BR rpc.gssd(8),
-- 
1.8.3.1

